src/i18n/pt-BR/index.js

```diff
@@ -6,85 +6,85 @@ export default {
     message: {
       ...potato,
       help: {
-        label: 'Summon Wizard'
+        label: 'Sumonar mago'
```

Comment:
Can we use something different like “Invocar mago” here? Sumonar doesn’t seem to be very widespread. What do you think?

Reply:
'Summon' tem tradução "convocar, invocar, chamar, citar". Acho que invocar seria mais interesante. Como trata-se de um programa, 'executar' também ficaria legal.